### PR TITLE
[15.0][MIG] sale_coupon_promotion_generate_coupon: Migration to version 15.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT42512

@chienandalu @stefan-tecnativa please review

### CODE REVIEW

**Review by chienandalu — COMMENTED** [review]
This is subject to module split

**Comment by pilarvargas-tecnativa** [comment]
> This is subject to module split

@chienandalu split done, please check again that everything is correct

**Review by chienandalu — APPROVED** [review]
Thanks @pilarvargas-tecnativa :)

please review @stefan-tecnativa

File: sale_coupon_promotion_generate_coupon/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Generate coupons in another coupon program in sales",
+    "summary": "Allows to generate pending coupons in a coupon program in sales",
+    "version": "15.0.1.0.0",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Sales Management",
+    "website": "https://github.com/OCA/sale-promotion",
+    "depends": ["sale_coupon"],
```
Comment (stefan-tecnativa) [inline sale_coupon_promotion_generate_coupon/__manifest__.py]: Shouldn't it depend on `coupon_promotion_generate_coupon` as well?

**Review by stefan-tecnativa — APPROVED** [review]
Functionallity working.

File: sale_coupon_promotion_generate_coupon/__manifest__.py (hunk 1 above)
Comment (chienandalu) [reply sale_coupon_promotion_generate_coupon/__manifest__.py]: Good catch. Please change it @pilarvargas-tecnativa

Comment (pilarvargas-tecnativa) [reply sale_coupon_promotion_generate_coupon/__manifest__.py]: thanks for the comment!

File: sale_coupon_promotion_generate_coupon/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Generate coupons in another coupon program in sales",
+    "summary": "Allows to generate pending coupons in a coupon program in sales",
+    "version": "15.0.1.0.0",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Sales Management",
+    "website": "https://github.com/OCA/sale-promotion",
+    "depends": ["coupon_promotion_generate_coupon"],
```
Comment (chienandalu) [inline sale_coupon_promotion_generate_coupon/__manifest__.py]: You can't remove the `sale_coupon` dependency
```suggestion
    "depends": ["sale_coupon", "coupon_promotion_generate_coupon"],
```

Comment (pilarvargas-tecnativa) [reply sale_coupon_promotion_generate_coupon/__manifest__.py]: thanks :)

**Review by chienandalu — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**